downstream/modules/platform/ref-controller-git-refspec.adoc

```diff
@@ -0,0 +1,28 @@
+[id="controller-git-refspec"]
+
+= Git Refspec
```

Comment:
Maybe the title should be 'The Git Refspec field', although the first sentence talks about the SCM Refspec field.